src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/WriteHelper.cs

```diff
@@ -397,25 +397,51 @@ public async Task<int> CopyEntityResourceAggregateHistory(ICollection<EntityReso
             await writer.WriteAsync(e.FromStateVersion, NpgsqlDbType.Bigint, token);
             await writer.WriteAsync(e.EntityId, NpgsqlDbType.Bigint, token);
             await writer.WriteAsync(e.FungibleResourceEntityIds.ToArray(), NpgsqlDbType.Array | NpgsqlDbType.Bigint, token);
+            await writer.WriteAsync(e.FungibleResourceLastUpdateStateVersions.ToArray(), NpgsqlDbType.Array | NpgsqlDbType.Bigint, token);
             await writer.WriteAsync(e.NonFungibleResourceEntityIds.ToArray(), NpgsqlDbType.Array | NpgsqlDbType.Bigint, token);
+            await writer.WriteAsync(e.NonFungibleResourceLastUpdateStateVersions.ToArray(), NpgsqlDbType.Array | NpgsqlDbType.Bigint, token);
         }
 
         await writer.CompleteAsync(token);
 
         return entities.Count;
     }
 
-    public async Task<int> CopyEntityResourceHistory(ICollection<EntityFungibleResourceHistory> fungibleEntities, ICollection<EntityNonFungibleResourceHistory> nonFungibleEntities, CancellationToken token)
+    public async Task<int> CopyEntityResourceVaultAggregateHistory(ICollection<EntityResourceVaultAggregateHistory> entities, CancellationToken token)
```

Comment:
I'm not sure we should call these methods `Copy`. What these methods do is write/save data into db. It's internal that it performs postgres copy operation.

Reply:
Going to change that as (part of) separate PR to ease up further merger to `release/betanet-v2`